api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.4.x.xml

```diff
@@ -28,14 +28,85 @@
+		<addForeignKeyConstraint constraintName="order_group_attribute_type_changed_by_fk" baseTableName="order_group_attribute_type" baseColumnNames="changed_by" referencedTableName="users" referencedColumnNames="user_id"/>
+		<addForeignKeyConstraint constraintName="order_group_attribute_type_retired_by_fk" baseTableName="order_group_attribute_type" baseColumnNames="retired_by" referencedTableName="users" referencedColumnNames="user_id"/>
+		<modifySql dbms="mssql">
+			<replace replace="CHAR(38)" with="UNIQUEIDENTIFIER DEFAULT NEWSEQUENTIALID()"/>
+		</modifySql>
+	</changeSet>
+	
+	<changeSet id="20200723-TRUNK-5410" author="tendomart">
+		<comment>Creating order_group_attribute table</comment>
+		<createTable tableName="order_group_attribute">
+			<column name="order_group_attribute_id" type="int" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false" />
+			</column>
+			<column name="order_group_id" type="int"><constraints nullable="false" /></column>
+			<column name="attribute_type_id" type="int"><constraints nullable="false" /></column>
+			<column name="value_reference" type="text"><constraints nullable="false" /></column>
+			<column name="uuid" type="char(38)"><constraints nullable="false" unique="true" /></column>
+			<column name="creator" type="int"><constraints nullable="false"/></column>
+			<column name="date_created" type="datetime"><constraints nullable="false"/></column>
+			<column name="changed_by" type="int"/>
+			<column name="date_changed" type="datetime" />
+			<column name="voided" type="boolean" defaultValueBoolean="false">
+				<constraints nullable="false"/>
+			</column>
+			<column name="voided_by" type="int" />
+			<column name="date_voided" type="datetime" />
+			<column name="void_reason" type="varchar(255)" defaultValue="null"/>
+		</createTable>
+		<addForeignKeyConstraint constraintName="order_group_attribute_order_group_fk" baseTableName="order_group_attribute" baseColumnNames="order_group_id" referencedTableName="order_group" referencedColumnNames="order_group_id" />
+		<addForeignKeyConstraint constraintName="order_group_attribute_attribute_type_id_fk" baseTableName="order_group_attribute" baseColumnNames="attribute_type_id" referencedTableName="order_group_attribute_type" referencedColumnNames="order_group_attribute_type_id" />
+		<addForeignKeyConstraint constraintName="order_group_attribute_creator_fk" baseTableName="order_group_attribute" baseColumnNames="creator" referencedTableName="users" referencedColumnNames="user_id" />
+		<addForeignKeyConstraint constraintName="order_group_attribute_changed_by_fk" baseTableName="order_group_attribute" baseColumnNames="changed_by" referencedTableName="users" referencedColumnNames="user_id" />
+		<addForeignKeyConstraint constraintName="order_group_attribute_voided_by_fk" baseTableName="order_group_attribute" baseColumnNames="voided_by" referencedTableName="users" referencedColumnNames="user_id" />
+	</changeSet>
```

Comment:
@tendomart have you run these changesets against the openmrs db to see if the tables are created with their respective fields ?, if yes then you can attach some screenshots

Reply:
@gitcliff  do you have a way of running the changesets apart from the OpenMRS-Bootsrapping process for an external Database?
Please share i want to learn how to do that.

About the screenshots please visit these links let me know if you find what you expect to see

[order_group_attribute_type](https://pasteboard.co/JkXDlTQ.png) Table

[order_group_attribute](https://pasteboard.co/JkXELHp.png)

Reply:
@tendomart the links are not showing anything ..
Abut running the changesets , you can run an openmrs instance on tomcat server having the war file that you generated during the `mvn clean install` build ,,,
then you can access your openmrs database via command line and see if the new tables are there with there fields and then take screen shots and share them

Reply:
> @tendomart the links are not showing anything ..
> Abut running the changesets , you can run an openmrs instance on tomcat server having the war file that you generated during the `mvn clean install` build ,,,
> then you can access your openmrs database via command line and see if the new tables are there with there fields and then take screen shots and share them

That's routine and i guess that's why the travis build passes with those changesets included , my question above is do you have any other way of running the changesets aside "mvn install / package  in the context of openmrs?

Reply:
i dont know of any other though what evidence or proof do you have that the tables have been created successfully in the openmrs db?

Reply:
most of the times when making changes in the db ie adding tables ,adding or removing fields from tables ,its advisable to attach some screenshots